FHEM/14_SD_WS_Maverick.pm

```diff
@@ -10,6 +10,7 @@
 # Version 1.1
 #  - changed: 14_SD_WS_Maverick: rename Readings for Temperatures
 #  - feature: 14_SD_WS_Maverick: added Readings for Sensor-states
+#  - feature: 14_SD_WS_Maverick: added support for device TFA 14.1504 (Radio-controlled grill and meat thermometer)
```

Comment:
Ergänzungsvorschlag:
changed: Removed internals which are identical to readings
changed: Optimized some code.